### PR TITLE
Use CSS variables over registerThemingParticipant - rulers viewparts

### PR DESCRIPTION
**Pull request by ghost**
#165169

### CODE REVIEW

**Comment by najmiehsa** [comment]
@microsoft-github-policy-service agree

File: src/vs/editor/browser/viewParts/rulers/rulers.css [hunk 1]
```diff
@@ -6,4 +6,5 @@
 .monaco-editor .view-ruler {
 	position: absolute;
 	top: 0;
-}
\ No newline at end of file
+	box-shadow: 1px 0 0 0 var(--vscode-editorRuler-ruler) inset !important;
```
Comment (rzhao271) [inline src/vs/editor/browser/viewParts/rulers/rulers.css:9]: Does `!important` need to be used here?

Comment (najmiehsa) [reply src/vs/editor/browser/viewParts/rulers/rulers.css:9]: I think it doesn't need to be overridden. I remove ``!important``

**Review by rzhao271 — APPROVED** [review]

**Review by lramos15 — APPROVED** [review]

**Comment by alexdima** [comment]
@lramos15 @rzhao271 This introduced a regression. See https://github.com/microsoft/vscode/pull/166698/files

**Pull request merged**